shim.js

```diff
@@ -0,0 +1,3 @@
+global.requestAnimationFrame = (callback) => {
```

Comment:
It might be useful adding a `TODO` and / or the link to the issue explaining this workaround (https://github.com/facebook/jest/issues/4545#issuecomment-333004504).